VirtoCommerce.CatalogModule.Data/Services/ItemServiceImpl.cs

```diff
@@ -116,6 +116,8 @@ public virtual void Delete(string[] itemIds)
                 .Select(i => new GenericChangedEntry<CatalogProduct>(i, EntryState.Deleted))
                 .ToList();
 
+            _commerceService.DeleteSeoForObjects(items);
```

Comment:
Decided to not add new method to this interface to avoid possible compatibility problems with existing custom implementations, so please use existing `DeleteSeoForObject`.